biggraphite/drivers/cassandra.py

```diff
@@ -1741,8 +1741,7 @@ def _get_metrics(self, metric_names):
         for (success, result) in results:
             if not success:
                 raise CassandraError("Failed to concurrently get metrics", result)
-            if result[0] is not None:
-                row = result[0]
+            for row in result:
```

Comment:
you probably want to check that `row` is not None.

Reply:
The row is returned by the ResultSet, I don't think it can be expected to be None.
When I look at the [documentation](https://datastax.github.io/python-driver/getting_started.html#executing-queries), I don't see examples checking for its nullity.

In the [accessor](https://github.com/criteo/biggraphite/blob/master/biggraphite/accessor.py#L517) we also iterate over the rows without checking for None.

Am I missing something?

Reply:
Hmm, I don't remember why we had this check in the first place .. you are right, it is probably fine

Reply:
Thanks!